core/comments/comment_view.ts

```diff
@@ -370,6 +377,27 @@ export class CommentView implements IRenderedElement {
   private truncateText(text: string): string {
     return text.length >= 12 ? `${text.substring(0, 9)}...` : text;
   }
+
+  private onDeleteDown(e: PointerEvent) {
```

Comment:
Can do, but I'll probably just give it the same format as blocks (`isDeadOrDying`) unless you have a different way you'd like it to look.

Reply:
That's fine. I just don't want to get to where we're inspecting whether a random SVG node is null as our way of checking for disposal >.<

Although that's no longer possible because of how you did initialization, so maybe it's fine to leave out `isDeadOrDying` and assume that we'll add it as soon as we have problems with events and disposal. Your call.